src/main/kotlin/uk/gov/dluhc/printapi/database/entity/AnonymousElectorDocument.kt

```diff
@@ -97,14 +97,14 @@ class AnonymousElectorDocument(
 ) {
 
     val status: AedPrintRequestStatus.Status?
-        get() = printRequests.sortedByDescending { it.requestDateTime }.first().status
+        get() = printRequests.maxBy { it.requestDateTime }.status
```

Comment:
Minor, @Neil-Massey. I think we can use the `getLatestPrintRequest` here to make the intent clearer and avoiding duplication of the logic.

<pre>
    val status: AedPrintRequestStatus.Status?
        get() = getLatestPrintRequest().status
</pre>